### ISSUE TITLE
Enhancement: Add plugin to be installed via UI

### ISSUE THREAD

**Issue by timonsku** [grafana/mqtt-datasource#47]
Adding it to the plugin registry so that it can be installed via the UI would be very useful.
There are a lot of dev dependencies and tools that need to be installed in order to build and install this plugin.
The process took me close to an hour on a Pi4 due to the sheer amount of dependencies. An easier way to install it given its an official plugin would be really great.

**Comment by wz2b** [comment]
plus, if it were in the catalog, it would be signed and you wouldn't have to add an unsigned exception

**Comment by JPFreeski** [comment]
Yeah how feasible is this. I'm struggling with how to get this up and running with docker-compose.

Any suggestions?
